WordPress/gutenberg
# AlignmentMatrixControl: keep the physical direction in RTL languages

Fix: #36237

## What?
~~This PR fixes a problem in the RTL language where tooltip labels for the `AlignmentMatrixControl` component are not displayed correctly.~~

This PR fixes the physical position of the grid in `AlignmentMatrixControl` component, regardless of language orientation.

## Why?
In the RTL language, the grid on which the `AlignmentMatrixControl` component is written is inverted left to right.
~~However, tooltip labels do not take language orientation into account, so the left-right labels will be reversed.~~
Therefore, the physical direction indicated by the tooltip is horizontally reversed in the RTL language.

## How?
~~In the variable that defines tooltip labels,  I added `isRTL` function to separate labels.~~
Applied `direction:ltr` to a row that is part of a component.
In addition, I have added a fix to maintain RTL language behavior in the cover block, which is the only one using this component.

## Testing Instructions

### Test on the storybook

- Build and open the storybook on this branch.
- In the `AlignmentMatrixControl` component, confirm that the grid is maintained when the language direction is changed and that the tooltip text also points in the correct direction.

### Test on the block editor

1. Build on `trunk` and open the post / site editor.
2. Insert a cover block and one block in it.
3. Use `AlignmentMatrixControl` to align to either left or right.
4. Use [RTL Tester plugin](https://wordpress.org/plugins/rtl-tester/) to change the language direction to RTL.
5. The blocks in the cover block are placed in reverse (The physical direction should be maintained, but is outside the scope of this issue.).
6. Ensures that the isRTL function returns `rtl` with the following code:

```
wp.hooks.addFilter( 'i18n.gettext_with_context', 'test', () => 'rtl' );
```

7. Confirm that the tooltip text shows the left and right sides reversed when the grid is focused.
8. Build on this branch and open the page.
9. Ensures that the isRTL function returns `rtl` with the following code:

```
wp.hooks.addFilter( 'i18n.gettext_with_context', 'test', () => 'rtl' );
```

10. In the cover block, confirm that the physical position of the child blocks is maintained and that the tooltip text also points in the correct direction.
